src/main/java/com/google/sps/data/Cache.java

```diff
@@ -14,8 +14,9 @@
 
   public GriddedData getGrid(Coordinates swCorner, Coordinates neCorner) {
     ArrayList<AQDataPoint> data = new ArrayList<>();
-    aqDataPointsPerDegree = 1000;
+    aqDataPointsPerDegree = 100;
```

Comment:
is 100 an arbitrary number?

Reply:
1000 was chosen originally to roughly estimate a 100m resolution, I scaled it down to 100 (which means it's a roughly 1km grid cell size) because it was taking too much time to create the grid.

Reply:
and because without implementing a dynamic grid scaling, the number of data points created was somewhat overwhelming for the heatmap visualiser, and made it sad.